sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor{TPartition}.cs

```diff
@@ -928,6 +927,11 @@ protected virtual LastEnqueuedEventProperties ReadLastEnqueuedEventProperties(st
                 ActivePartitionProcessors.Clear();
                 _runningProcessorTask = RunProcessingAsync(_runningProcessorCancellationSource.Token);
             }
+            catch (OperationCanceledException ex)
+            {
+                Logger.EventProcessorStartError(Identifier, EventHubName, ConsumerGroup, ex.Message);
+                throw new TaskCanceledException();
```

Comment:
Do you think it would be worthwhile to set the inner exception here, to the `OperationCanceledException` we just caught?

Reply:
I thought about it, but I haven't seen the case where it provides any relevant detail and we want to be sure not to give the impression that the [Event Hubs Troubleshooting](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-messaging-exceptions) guidance applies in this case.

Maybe you're thinking of a scenario that I'm overlooking.... anything specific in mind?

Reply:
> Maybe you're thinking of a scenario that I'm overlooking.... anything specific in mind?

Nope, what you said makes sense here.